pkg/gcc/send_side_bwe.go

```diff
@@ -176,6 +184,16 @@ func (e *SendSideBWE) OnTargetBitrateChange(f func(bitrate int)) {
 	e.onTargetBitrateChange = f
 }
 
+// IsClosed returns true if SendSideBWE is closed
+func (e *SendSideBWE) IsClosed() bool {
```

Comment:
Do we need to export `IsClosed` or is it enough to have it in this package?

Reply:
Thinking about it again, it should be enough to keep it inside the package as other `isClosed` implementations are also not exposed.